static/src/javascripts/projects/commercial/modules/dfp/Advert.ts

```diff
@@ -14,48 +14,22 @@ type Timings = {
 	lazyWaitComplete: number | null;
 };
 
-const stringToTuple = (size: string): AdSizeTuple => {
-	const dimensions = size.split(',', 2).map(Number);
-
-	// Return an outOfPage tuple if the string is not `{number},{number}`
-	if (dimensions.length !== 2 || dimensions.some((n) => isNaN(n)))
-		return [0, 0]; // adSizes.outOfPage
-
-	return [dimensions[0], dimensions[1]];
+const isSlotName = (slotName: string): slotName is SlotName => {
+	return slotName in slotSizeMappings;
 };
 
-/** A breakpoint can have various sizes assigned to it. You can assign either on
- * set of sizes or multiple.
- *
- * One size       - `data-mobile="300,50"`
- * Multiple sizes - `data-mobile="300,50|320,50"`
- */
-const createSizeMapping = (attr: string): AdSize[] =>
-	attr
-		.split('|')
-		.map((size) => (size === 'fluid' ? 'fluid' : stringToTuple(size)));
-
-/** Extract the ad sizes from the breakpoint data attributes of an ad slot
- *
- * @param advertNode The ad slot HTML element that contains the breakpoint attributes
- * @returns A mapping from the breakpoints supported by the slot to an array of ad sizes
- */
-const getAdBreakpointSizes = (advertNode: HTMLElement): AdSizes =>
-	breakpoints.reduce<Record<string, AdSize[]>>((sizes, breakpoint) => {
-		const data = advertNode.getAttribute(
-			`data-${breakpointNameToAttribute(breakpoint.name)}`,
-		);
-		if (data) {
-			sizes[breakpoint.name] = createSizeMapping(data);
-		}
-		return sizes;
-	}, {});
+const getAdSizeMapping = (name: string): SizeMapping | undefined => {
+	const slotName = /inline\d+/.test(name) ? 'inline' : name;
```

Comment:
Just wondering if we need a regex? Would this simplification suffice:

```suggestion
	const slotName = name.includes('inline') ? 'inline' : name;
```

Reply:
good point, I don't think another slot is going to come along with `inline` in the name!